plop-templates/@react-aria/package.json.hbs

```diff
@@ -7,6 +7,11 @@
   "main": "dist/main.js",
   "module": "dist/module.js",
   "types": "dist/types.d.ts",
+  "exports": {
+    "types": "dist/types.d.ts",
+    "import": "dist/module.js",
```

Comment:
Small thing: these should be 
```suggestion
    "import": "dist/module.mjs",
```
right? Same issue for the other plop templates if so

Reply:
ugh, yep, i'll do here so we can back out if we need to easily